pkg/arvo/app/metadata-store.hoon

```diff
@@ -61,9 +61,27 @@
     ^-  (quip card _this)
     ?>  (team:title our.bowl src.bowl)
     =^  cards  state
-      ?:  ?=(%metadata-action mark)
+      ?+  mark  [~ state]
```

Comment:
Shouldn't this fall back to `+on-poke:def` instead of a direct no-op?

Reply:
Fixed, thanks,